src/table/columnCheckboxes.component.tsx

```diff
@@ -1,29 +1,20 @@
 import React from 'react';
-import { Column } from 'react-table';
 import { Checkbox } from '@mui/material';
 import { useAppDispatch, useAppSelector } from '../state/hooks';
 import {
-  selectSelectedColumns,
+  selectSelectedChannels,
   selectColumn,
   deselectColumn,
 } from '../state/slices/tableSlice';
 import { useVirtualizer } from '@tanstack/react-virtual';
-import { useAvailableColumns } from '../api/channels';
+import { useChannels } from '../api/channels';
+import { FullChannelMetadata } from '../app.types';
 
 const ColumnCheckboxes = React.memo((): React.ReactElement => {
-  const { data: availableColumns } = useAvailableColumns();
+  const { data: channels } = useChannels();
```

Comment:
I actually prefer that after the refactor, we can just use `useChannels` here and it makes the code in here a lot simpler

Reply:
Okay hold up a sec, see my earlier comment. Do we want the columnCheckboxes to be passed the availableColumns or the channels from recordTable or what exactly?

Reply:
I mean it doesn't matter really as `columnCheckboxes` is a test component that won't end up in production, but it doesn't really matter which one it uses. `useChannels` makes the code a little nicer imo? It's the same underlying query, just that `useAvailableColumns` transforms it into `Columns`. But we don't need `Column` objects in the checkboxes - we just need a list of IDs essentially

Reply:
Right I'm with you, I think I can tackle this all in my head now!